config.md

```diff
@@ -132,7 +132,7 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
 * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page.
+* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
```

Comment:
If we're looking for a clear way to specify the “relevant kernel interface”, [`cap_from_text(3)` is part of the withdrawn POSIX.1e draft specification][1] (and therefore has some chance of being cross-platform) and is supported by [libcap][2].  Without clarification, I'd expect the kernel intefaces to be [`capset(2)`][3] and/or [`prctl(2)`][4], but both of those happen after the capabilities have been converted to integers.

[1]: http://www.man7.org/linux/man-pages/man3/cap_from_text.3.html#CONFORMING_TO
[2]: https://git.kernel.org/pub/scm/linux/kernel/git/morgan/libcap.git/
[3]: http://www.man7.org/linux/man-pages/man2/capset.2.html
[4]: http://www.man7.org/linux/man-pages/man2/prctl.2.html